.vscode/launch.json

```diff
@@ -1,36 +1,35 @@
 {
-    // Use IntelliSense to learn about possible attributes.
```

Comment:
This shouldnt be checked into source control.   I would add this to the .gitignore.